lib/Extension/Core/ColumnType/DateTime.php

```diff
@@ -186,11 +191,11 @@ private function getInputData($value)
                     );
                 }
 
-                    if (empty($value)) {
-                        $inputData[$field] = null;
-                    } else {
-                        $inputData[$field] = $this->transformStringToDateTime($value, $mappingFormat);
-                    }
+                if (empty($value)) {
+                    $inputData[$field] = null;
+                } else {
+                    $inputData[$field] = $this->transformStringToDateTime($value, $mappingFormat);
```

Comment:
`$inputData[$field] = !empty($value) ? $this->transformStringToDateTime($value, $mappingFormat) : null;`